### PR TITLE
Add ability to dockerize kubelet in local cluster

### PR DESCRIPTION
**Pull request by pmorie**
@vmarmol

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -141,16 +148,31 @@ sudo -E "${GO_OUT}/kube-controller-manager" \
...
+  --volume=/:/rootfs:ro \
+  --volume=/var/run:/var/run:rw \
+  --volume=/sys:/sys:ro \
+  --volume=/var/lib/docker/:/var/lib/docker:ro \
+  --volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
+  --net=host \
+  --detach=true \
+  --privileged=true \
+  gcr.io/google_containers/kubelet:v0.16 \
+  /kubelet --v=3 --containerized --chaos-chance="${CHAOS_CHANCE}" --hostname-override="127.0.0.1" --address="127.0.0.1" --api-servers="${API_HOST}:${API_PORT}" --port="$KUBELET_PORT" --resource-container="")
```
Comment (dchen1107) [inline hack/local-up-cluster.sh]: Can we redirect kubelet logging to a log file, just like what we did for master components?

**Comment by dchen1107** [comment]
LGTM thanks for all your effort on running kubelet in a container.

**Comment by pmorie** [comment]
Depends on #7797

**Comment by vmarmol** [comment]
@pmorie looks like we need a rebase

**Comment by pmorie** [comment]
@vmarmol rebased

**Comment by vmarmol** [comment]
I am sorry to say...but looks like we'll need another rebase :P

**Comment by pmorie** [comment]
NP

**Comment by pmorie** [comment]
K, the kubelet output goes to `/tmp/kubelet.log` now

**Pull request merged**